src/components/elections-stats/elections-stats.ce.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div class="flex flex-col">
+    <ElectionsGraph
+      v-for="graph in graphs"
+      :key="graph.title"
+      :data="graph.data"
+      :title="graph.title"
+      :id="graph.id"
+    />
+  </div>
+</template>
+
+<script>
+import axios from "../../_common/axios.mjs";
+import ElectionsGraph from "../elections-graph/elections-graph.ce.vue";
+export default {
+  name: "ElectionsStats",
+  components: {
+    ElectionsGraph,
+  },
+  props: {
+    title: {
+      type: String,
+      default: "",
+    },
+    sports: {
+      type: Boolean,
+      default: false,
+    },
+    societies: {
+      type: Boolean,
+      default: false,
+    },
+    departments: {
+      type: Boolean,
+      default: false,
+    },
+    accommodations: {
+      type: Boolean,
+      default: false,
+    },
+    generalStats: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      all: true,
+      selected: [],
+      graphs: [],
+      stats: [],
+    };
+  },
+  mounted() {
+    this.getStatistics();
+  },
+  methods: {
+    async getStatistics() {
+      let self = this;
+      await axios
+        .get("https://pluto.sums.su/api/elections/statistics")
+        .then(function (response) {
+          console.log(response.data);
+          if (self.sports) {
+            var sports = response.data.activities_subcats.find(
+              (item) => item.category_data.name === "Sports",
+            ).activity_data;
+            sports = sports.filter((item) => item.total_members > 15);
```

Comment:
Can we make the number of members to remove if it's below to be a variable so we can configure? Maybe a prop with a default value of 15. This can then be updated in the other places we use it on this file.